pkg/persistence/config/loader/config_loader_test.go

```diff
@@ -549,11 +591,32 @@ configs:
       schemaVersion: '1.0'
       scope: 'environment'
       insertAfter:
-        type: reference
-        configId: configId
-        property: name
-        configType: something`,
-			wantErrorsContain: []string{`failed to parse insertAfter: property field of reference parameter "project:something:configId:name" must be "id"`},
+        type: environment
+        name: ENV_VAR_SOME_RANDOM_VAR
+`,
+			wantConfigs: []config.Config{
+				{
+					Coordinate: coordinate.Coordinate{
+						Project:  "project",
+						Type:     "builtin:profile.test",
+						ConfigId: "profile-id",
+					},
+					Type: config.SettingsType{
+						SchemaId:      "builtin:profile.test",
+						SchemaVersion: "1.0",
+					},
+					Template: template.NewInMemoryTemplate("profile.json", "{}"),
+					Parameters: config.Parameters{
+						config.NameParameter:        &value.ValueParameter{Value: "Star Trek > Star Wars"},
```

Comment:
Star Trek < Star Wars

Reply:
This is factually very wrong. Star Trek is Star Wars light years ahead.